File: PTApi/Data/Models/Kelas.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace PTApi.Data.Models
{
    public class Kelas
    {
        [Key]
        [StringLength(5)]
        public string KodeKelas { get; set; }
        public string Nama { get; set; }

        public List<MhsKls> Mhss { get; set; }
        public Dosen Dosen { get; set; }
    }
}

File: PTUI/src/app/model/kelas.ts
import { Mahasiswa } from "./mahasiswa";
import { Dosen } from "./dosen";

export class Kelas {
    constructor(
        public KodeKelas: string,
        public Nama: string,
        public Dosen: Dosen[],
        public Mhss: Mahasiswa[],
        public NilaiMid: string,
        public NilaiSem: string,
        public NilaiMutu: string
    ) { }
}